apps/src/code-studio/components/progress/lessonLockDialog/LessonLockDialog.jsx

```diff
@@ -23,7 +23,8 @@ function LessonLockDialog({
   lessonId,
   handleClose,
   selectedSectionId,
-  refetchSectionLockStatus
+  refetchSectionLockStatus,
+  isHidden
```

Comment:
nit: I think it would be if it this prop were named `lessonIsHidden` or something. `isHidden` sounds like it applies to the dialog itself

Reply:
Good call- I've updated this to lessonIsHidden for tests and lessonLockDialog, though I kept it as 'isHidden' further up the data pipeline so it matches in places like ProgressLessonTeacherInfo.